diff --git a/lib/neutron_plugins/midonet b/lib/neutron_plugins/midonet
index 17a612a..78e8b1d 100644
--- a/lib/neutron_plugins/midonet
+++ b/lib/neutron_plugins/midonet
@@ -27,6 +27,8 @@ function neutron_plugin_configure_common {
     Q_PLUGIN_CONF_PATH=etc/neutron/plugins/midonet
     Q_PLUGIN_CONF_FILENAME=midonet.ini
     Q_PLUGIN_CLASS="midonet.neutron.plugin.MidonetPluginV2"
+
+    iniset $NEUTRON_CONF DEFAULT api_extensions_path $MIDONET_DIR/python-neutron-plugin-midonet/midonet/neutron/extensions
 }

 function neutron_plugin_configure_debug_command {
