diff --git a/lib/neutron_plugins/midonet b/lib/neutron_plugins/midonet
index c5373d6..95ca741 100644
--- a/lib/neutron_plugins/midonet
+++ b/lib/neutron_plugins/midonet
@@ -27,7 +27,7 @@ function neutron_plugin_configure_common {
     Q_PLUGIN_CONF_PATH=etc/neutron/plugins/midonet
     Q_PLUGIN_CONF_FILENAME=midonet.ini
     Q_DB_NAME="neutron_midonet"
-    Q_PLUGIN_CLASS="neutron.plugins.midonet.plugin.MidonetPluginV2"
+    Q_PLUGIN_CLASS="midonet.neutron.plugin.MidonetPluginV2"
 }

 function neutron_plugin_configure_debug_command {
