diff --git a/stack.sh b/stack.sh
index ec13338..800d1e7 100755
--- a/stack.sh
+++ b/stack.sh
@@ -1277,6 +1277,10 @@ fi
 # Once neutron agents are started setup initial network elements
 if is_service_enabled q-svc; then
     echo_summary "Creating initial neutron network elements"
+    iniset $MIDONET_PLUGIN_SRC_DIR/midonet/neutron/db/migration/alembic.ini alembic sqlalchemy.url `database_connection_url $Q_DB_NAME`
+    cd $MIDONET_PLUGIN_SRC_DIR/midonet/neutron/db/migration/
+    midonet-db-manage upgrade head
+    cd -
     create_neutron_initial_network
     setup_neutron_debug
 fi
