diff --git a/lib/neutron_thirdparty/midonet b/lib/neutron_thirdparty/midonet
index 099a66e..220718e 100644
--- a/lib/neutron_thirdparty/midonet
+++ b/lib/neutron_thirdparty/midonet
@@ -25,7 +25,21 @@ function configure_midonet {
 }

 function init_midonet {
-    :
+   # patched by midostack until we migrate to a new API, that does
+   # create the provider router under the hood.
+   ADMIN_TENANT_ID=$(openstack project show admin | grep " id " | get_field 2)
+    cat <<EOF > ~/.midonetrc
+[cli]
+username=admin
+api_url=http://localhost:8080/midonet-api/
+project_id=admin
+tenant=$ADMIN_TENANT_ID
+password=$ADMIN_PASSWORD
+EOF
+
+    provider_router_id=$(PYTHONPATH=/opt/stack/midonet/python-midonetclient/src/ /opt/stack/midonet/python-midonetclient/src/bin/midonet-cli -e router create name MidonetProviderRouter)
+
+    iniset /etc/neutron/plugins/midonet/midonet.ini MIDONET provider_router_id $provider_router_id
 }

 function install_midonet {
